Bottom: 9900e049a2cfb61eca25f2b73593b9d6b31d6f3a
Top:    00471f9341df4d23ea9eed47ae225ee50f69e556
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 00:32:37 +0000

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index ffdca7f..2d02908 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -997,10 +997,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
